lib/version.d.ts

```diff
@@ -0,0 +1,20 @@
+export declare const VERSION_PATTERN: string;
+export declare function valid(version: string): string | null;
+export declare function clean(version: string): string | null;
+export declare function stringify(version: string): string | null;
+
+interface parsed {
```

Comment:
Do not duplicate, use a shared types file and import from it.

Reply:
Done🟩
Added types for functions with similar parameters as well, needs better naming though.